app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,46 @@
+class TeamsController < ApplicationController
+  authorize_persona class_name: "User"
+  before_action :validate_already_existing_user, only: [:create]
+  before_action :same_user, only: [:create, :new, :index]
+  grant(
+    manager: :all
+  )
+
+  def index
+    @team = current_user.team_members
+  end
+
+  def new
+  end
+
+  def create
+    if current_user.max_team_members_reached?
+      flash[:alert] = 'You are not allow to add more members to your team!'
+    else
+      User.invite! email: params[:email],
+                  first_name: params[:first_name],
+                  last_name: params[:last_name],
+                  authorization_tier: :user,
+                  manager: current_user
+      flash[:notice] = "The invitation was send to #{params[:email]}"
+    end
+    redirect_back(fallback_location: root_path)
+  end
+
+  private
+
+  def validate_already_existing_user
+    user_found = User.find_by(email: params[:email])
```

Comment:
same here. how may we know these are permitted or sanitized?

Reply:
I think the same applies here.